### DISCUSSION TITLE
crosshair color change and add add extra info

### DISCUSSION THREAD

**Discussion by armeya25** [highfestiva/finplot#433] [Q&A] [answered]
i change background color to black,
now tooltip/crosshair doesn't shows as its black color

how do i change its default color
also wish to add open, close, high, low, date info to be display there is there way to do it

**Comment by armeya25** [comment]
fplt.cross_hair_color = "#eef"

now i wants to add ohlc data on crosshair

**Comment by highfestiva** [comment, marked as answer]
See examples, I think it's in the S&P 500 one.

**Reply by armeya25** [reply]
its works in example,
dont work on my code,
Inspection error: <class 'IndexError'> single positional indexer is out-of-bounds

anyway thanks for reply.
i wont be using that
